src/style.css

```diff
@@ -1 +1,42 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  gap: 0;
```

Comment:
The 'gap' property is not applicable to the body element. It is used for grid and flex containers. If you intended to set the distance between elements, use 'margin' or 'padding' on those elements instead. This is a code quality issue.